### PR TITLE
add Modus Create

### PR DESCRIPTION
**Pull request by shoesheill**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @shoesheill, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 15c5490).  See below for details:

```
README.md: Company is listed out of order: "Mobile Jazz" (should be before "Modus Create")
moduscreate.md: Company title "Modus Create" doesn't match filename (expected ~ "modus-create.md")

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @shoesheill, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 3d65d8f). :heavy_check_mark:

**Comment by shoesheill** [comment]
Updated

**Comment by nylen** [comment]
@shoesheill did you close this PR on purpose? Please reopen it and we will be happy to merge.

File: company-profiles/modus-create.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Remote status
+
+Remote jobs are very hard to land. At Modus, most of our team is remote.
+While the majority of our team is remote, the volume of job openings we have is somewhat limited and very sporadic. That being said, each remote job we post gets hundreds and hundreds of applicants from every corner of the globe.
+
+## Region
+### Countries:
+Argentina, Brazil, Bulgaria, Canada, Chile, Croatia, England, Egypt, Germany, Guatemala, India, Lithuania, Macedonia, Mexico, Moldova, Nepal, Romania, Russia, Serbia, Spain, Sri Lanka, Sweden, Thailand, Ukraine, Uruguay
+### States: VA, NY, CA, CT, TX, MO, RI, MD, WY
```
Comment (nylen) [inline company-profiles/modus-create.md]: Let's change this part as follows:
```
### US States
VA, NY, CA, CT, TX, MO, RI, MD, WY
```

**Pull request closed without merging**